hackathon_site/dashboard/frontend/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
This is going to fail - can you put a really basic test in that works? Something like:

```javascript
test('renders placeholder text', () => {
  const { getByText } = render(<App />);
  const textElement = getByText(/IEEeeeeee/i);
  expect(textElement).toBeInTheDocument();
});
```

I think it's fine to flesh out the tests here later on.